index.html

```diff
@@ -454,11 +454,11 @@
       <td>
       <div class="container">
         <h2>Bertram Vidgen</h2>
-        <p class="title">PhD student or Enrichment student</p>
+        <p class="title">Enrichment student</p>
         <h3>Research interests</h3>
-        <p class="research_interests">List of research interests separated by a semi-colon</p>
```

Comment:
Put text between <p> and </p> not in class = ""

Reply:
leaving a comment